tests/failpoints/cases/test_metrics_overflow.rs

```diff
@@ -0,0 +1,21 @@
+// Copyright 2022 TiKV Project Authors. Licensed under Apache-2.0.
+
+use test_raftstore::*;
+
+#[test]
+fn test_memory_metrics_overflow() {
```

Comment:
The test can pass without the fix.

Reply:
Cannot, keep the store loop recv and handle msgs before the metrics add, then the test must fail.